### PR TITLE
13.0 opw 2261134 imp agr

### PR DESCRIPTION
**Pull request by agr-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by agr-odoo** [comment]
@nim-odoo 
I guess 2 separate commits are better, maybe 2 separate pr also?

**Comment by fclementic2c** [comment]
:)

**Comment by nim-odoo** [comment]
robodoo rebase-ff

File: addons/account/models/account_reconcile_model.py [hunk 1]
```diff
@@ -665,10 +665,13 @@ def _check_rule_propositions(self, statement_line, candidates):
                 total_residual += aml['aml_currency_id'] and aml['aml_amount_currency'] or aml['aml_balance']
             else:
                 total_residual += aml['aml_currency_id'] and aml['aml_amount_residual_currency'] or aml['aml_amount_residual']
+            total_currency = aml['aml_currency_id'] and self.env['res.currency'].browse(aml['aml_currency_id']) or self.company_id.currency_id
```
Comment (nim-odoo) [inline addons/account/models/account_reconcile_model.py:668]: Are you sure all `candidates` have the same currency?

Comment (agr-odoo) [reply addons/account/models/account_reconcile_model.py]: nice, I was testing with just a single candidate :)

**Comment by nim-odoo** [comment]
@fclementic2c @grindtildeath Does that seem to fix all you use cases?

**Comment by grindtildeath** [comment]
@nim-odoo @agr-odoo 
From my point of view, the change looks good. :+1: 
Thank you :slightly_smiling_face:

**Comment by smetl** [comment]
@nim-odoo @agr-odoo Guys... ask a review from the accounting team next time. Your are replacing some bug by anothers...

**Pull request closed without merging**